tests/requirements.txt

```diff
@@ -1 +1,4 @@
-nose==1.3.7
\ No newline at end of file
+nose==1.3.7
+pytest
```

Comment:
maybe `pytest`, `flake8` and `autopep8` should be removed, since they're dependencies of my IDE and autoformatting preferences.

Reply:
I think they are probably helpful for development so worth keeping